backend/tests/integration/orcid/test_orcid_cli_import_legacy_tokens.py

```diff
@@ -1,564 +0,0 @@
-#
-# Copyright (C) 2019 CERN.
-#
-# inspirehep is free software; you can redistribute it and/or modify it under
-# the terms of the MIT License; see LICENSE file for more details.
-
-import pytest
-from flask import current_app
-from helpers.factories.db.invenio_records import TestRecordMetadata
-from helpers.utils import get_index_alias
-from inspirehep.orcid import push_access_tokens
-from inspirehep.orcid.tasks import (
-    USER_EMAIL_EMPTY_PATTERN,
-    _find_user_matching,
-    _register_user,
-    legacy_orcid_arrays,
```

Comment:
could you please also remove the `legacy_orcid_arrays?` and the `cli`? 🚜

Reply:
Done!